testing_and_setup/compass/ocean/jigsaw_to_MPAS/build_mesh.py

```diff
@@ -21,7 +20,7 @@
 from mpas_tools.conversion import convert
 from mpas_tools.io import write_netcdf
 
-
+from jigsaw_to_MPAS.jigsaw_driver import jigsaw_driver
```

Comment:
@pwolfram, the import is here

Reply:
Thanks @sbrus89! I blanked it out because of the white space line above.  I'd recommend remove line 23/22 for clarity because typically all imports are lumped together.

Reply:
Some folks leave the space-- this is ok.

Reply:
I like to separate external packages from local ones in imports. @sbrus89 was using my code, so that was my call.

Reply:
I like that approach @xylar, so I'll leave it